test/inductor/test_torchinductor.py

```diff
@@ -199,9 +216,15 @@ def upcast_fn(x):
             else:
                 return x
 
+        def get_original_lowp_dtype(example_inputs):
+            dtypes = [x.dtype for x in example_inputs if isinstance(x, torch.Tensor)]
+            dtype_set = set(dtypes)
+            return dtype_set.pop() if len(dtype_set) == 1 else torch.half
```

Comment:
This is used to support bfloat16 type? What if some of the inputs are bfloat16 (or half) and some are float?